### ISSUE TITLE
Build issues with version 0.20.0

### ISSUE THREAD

**Issue by constantinpape** [QuantStack/xtensor-python#166]
I am having build issues wit the latest release of xtensor-python (0.20.0)
and xtensor (0.17.4) / xtl (0.4.16).

The lines 
```
#define FORCE_IMPORT_ARRAY
#include "xtensor-python/pyarray.hpp"
```
are causing: 

```
                 from /home/cpape/Work/software/src/z5/src/python/lib/z5py.cxx:7:
/home/cpape/Work/software/conda/miniconda3/envs/z5-bld/include/xtensor/xiterator.hpp: In function ‘constexpr auto xt::detail::trivial_begin(C&)’:
/home/cpape/Work/software/conda/miniconda3/envs/z5-bld/include/xtensor/xiterator.hpp:406:25: error: ‘xtl::mpl’ has not been declared
             return xtl::mpl::static_if<has_storage_iterator<C>::value>([&](auto self)
                         ^~~
/home/cpape/Work/software/conda/miniconda3/envs/z5-bld/include/xtensor/xiterator.hpp: In function ‘constexpr auto xt::detail::trivial_end(C&)’:
/home/cpape/Work/software/conda/miniconda3/envs/z5-bld/include/xtensor/xiterator.hpp:418:25: error: ‘xtl::mpl’ has not been declared
             return xtl::mpl::static_if<has_storage_iterator<C>::value>([&](auto self)
                         ^~~
/home/cpape/Work/software/conda/miniconda3/envs/z5-bld/include/xtensor/xiterator.hpp: In function ‘constexpr auto xt::detail::trivial_begin(const C&)’:
/home/cpape/Work/software/conda/miniconda3/envs/z5-bld/include/xtensor/xiterator.hpp:430:25: error: ‘xtl::mpl’ has not been declared
             return xtl::mpl::static_if<has_storage_iterator<C>::value>([&](auto self)
                         ^~~
/home/cpape/Work/software/conda/miniconda3/envs/z5-bld/include/xtensor/xiterator.hpp: In function ‘constexpr auto xt::detail::trivial_end(const C&)’:
/home/cpape/Work/software/conda/miniconda3/envs/z5-bld/include/xtensor/xiterator.hpp:442:25: error: ‘xtl::mpl’ has not been declared
             return xtl::mpl::static_if<has_storage_iterator<C>::value>([&](auto self)
                         ^~~
```

I am using Ubuntu 18.04 and gcc 7.3.

Note that this error does not occur with `xtensor-python` (0.19.1), `xtensor` (0.16.4) and `xtl` (0.4.16)

**Comment by JohanMabille** [comment]
Thanks for reporting!
The following workaround should fix your issue, we'll release a version with a fix soon.

```cpp
#include "xtl/xmeta_utils.hpp"
ine FORCE_IMPORT_ARRAY
#include "xtensor-python/pyarray.hpp"
```

**Comment by constantinpape** [comment]
The workaround fixed it, thanks!

**Comment by JohanMabille** [comment]
We have released 0.20.1 with a fix for this issue.
